modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/OperationReply.java

```diff
@@ -579,11 +579,15 @@ public SequenceTag buildSequence(Operation operation, SequenceTag sequenceElemen
                 if (policy.getCode().equals(PluginConstants.OperationCodes.PASSCODE_POLICY)) {
                     AtomicTag atomicTagElement = new AtomicTag();
                     List<AddTag> addConfig;
+                    DeleteTag deleteTag = new DeleteTag();
                     try {
                         addConfig = appendAddInfo(policy);
                         atomicTagElement.setAdds(addConfig);
                         atomicTagElement.setCommandId(operation.getId());
-
+                        List<Item> deleteItems = buildDeletePassCode(policy);
```

Comment:
deleteTag is a class in syncml engine.Refactored deleteItems to deleteTagItems from https://github.com/wso2/product-mdm/pull/156